docs/maps/mapLine-series.md

```diff
@@ -0,0 +1,30 @@
+Map line series
+===============
+
+A `mapLine` series is a special case of the map series where the value colors are applied to the strokes rather than the fills. 
+It can also be used for freeform drawing, like separators, in the map.
+
+<iframe style="width: 100%; height: 520px; border: none;" src="https://highcharts.com/samples/embed/maps/demo/mapline-mappoint" allow="fullscreen"></iframe>
+
+Configuration
+-------------
+
+To create a `mapLine` series it is necessary to set the series `type` as [mapline](https://api.highcharts.com/highmaps/series.mapline).
+More information about data formatting and available options can be found in the [API](https://api.highcharts.com/highmaps/series.mapline.data). 
+As you can see in the demo above we can create `mapLine` series in combination with a standard `map` series, then plot our shape data via the [path](https://api.highcharts.com/highmaps/series.mapline.data.path) option. This opens up many possibilities for the mentioned drawing of non-standard shapes.
```

Comment:
> then plot our shape data via the [path]

We should also mention `geometry` which are GeoJSON and TopoJSON compliant. As we're moving towards client-side projection, more focus should be on geometry. The `path` option in the future will only be used for non-geographic maps like schematics, house drawings etc.